### DISCUSSION TITLE
autogen.sh not working on Windows 10

### DISCUSSION THREAD

**Discussion by ahmedosama07** [libusb/libusb#1391] [Q&A]
```
Can't locate Autom4te/ChannelDefs.pm in @INC (you may need to install the Autom4te::ChannelDefs module) (@INC contains: /mingw/share/autoconf /usr/lib/perl5/site_perl /usr/share/perl5/site_perl /usr/lib/perl5/vendor_perl /usr/share/perl5/vendor_perl /usr/lib/perl5/core_perl /usr/share/perl5/core_perl) at /c/MinGW/bin/autoreconf-2.68 line 40.
BEGIN failed--compilation aborted at /c/MinGW/bin/autoreconf-2.68 line 40
```

When trying to run autogen.sh it gives the following error

**Comment by mcuee** [comment]
Please use MSYS2 mingw64 or mingw32 distribution. 
https://www.msys2.org/

The 32bit only [MinGW distribution](https://sourceforge.net/projects/mingw/) is not really supported but patches might get accepted if they are not too intrusive.

**Reply by ahmedosama07** [reply]
I tried with MSYS2 mingw64 and it gives this 
```
./bootstrap.sh: line 10: exec: autoreconf: not found
```

**Reply by mcuee** [reply]
You have to install mingw64 version of autoconf, automake and libtools.
